packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx

```diff
@@ -24,6 +24,7 @@ import { getSection } from './section/section-helper';
 import { cancelRegistration, parseAddressTemplateXml, scrollIntoView } from './patient-registration-utils';
 import { useInitialAddressFieldValues, useInitialFormValues, usePatientUuidMap } from './patient-registration-hooks';
 import { ResourcesContext } from '../offline.resources';
+import { add } from 'lodash-es';
```

Comment:
I think that this is unused and can be removed (according to a simple CTRL+F) - otherwise I'd import it via `import add from 'lodash-es/add';`.

Reply:
Ohh I added this unintentionally. I'll just remove this too.